cli/bench/main.rs

```diff
@@ -496,6 +502,36 @@ async fn main() -> Result<()> {
     new_data.max_latency = max_latency;
   }
 
+  if benchmarks.contains(&"runtimes") && cfg!(not(target_os = "windows")) {
+    runtimes::setup();
+    let stats = runtimes::ssr()?;
+    let req_per_sec = stats
+      .iter()
+      .map(|(name, result)| (name.clone(), result.requests as i64))
+      .collect();
+    reporter.write("runtimes_ssr_rps", &req_per_sec);
+    new_data.runtimes_ssr_rps = req_per_sec;
+    
+    let max_latency = stats
+      .iter()
+      .map(|(name, result)| (name.clone(), result.latency))
+      .collect();
+    reporter.write("runtimes_ssr_max_latency", &max_latency);
+    new_data.runtimes_ssr_max_latency = max_latency;
+
+    let sqlite = runtimes::sqlite()?;
+    for (name, data) in sqlite.iter() {
+      reporter.write_one("runtimes_sqlite", name, &data);
+    }
+    new_data.runtimes_sqlite = sqlite;
+
+    let ffi = runtimes::ffi()?;
```

Comment:
How do I run the ffi benchmarks for only deno? 

something like?
```
cargo bench --bench deno_bench -- ffi deno
```

Reply:
`See cli/bench/runtimes/README.md`; you'd do `cd cli/bench/runtimes && deno task setup && deno task ffi:deno`